proto/iot.proto

```diff
@@ -119,6 +126,34 @@ message Thermostat {
     Double cooling_setpoint = 11;
 }
 
+//Schedule of one week
+message Schedule {
```

Comment:
Can we clarify the name of this message (and its submessages) to be specific that it is a thermostat schedule? Something like `ThermostatSchedule` should be fine.

If we want, we might want to consider having a `Schedule` "wrapper object" sort of like [XBOSIotDeviceState](https://github.com/gtfierro/xboswave/blob/master/proto/iot.proto#L44-L61), which contains pointers to common structures but maintains common fields in the wrapper (like the timestamp).

@jhkolb what do you think?

Reply:
Is your thinking here that we might eventually have multiple notions of a schedule, for different devices? If we think that's a likely scenario, then a wrapper object sounds like a good idea.

Reply:
Yeah, the `Schedule` wrapper object would be where we would collect some "standard" XBOS definitions of different device schedules

Reply:
Makes sense, I added the "Thermostat" prefix to all of the new messages. I left out the main "Schedule" struct for now. I think the idea makes sense, but maybe we can introduce it a bit later when we populate the iot.proto file with more different kinds of device schedules?

Reply:
Thanks! Sounds good to me. I can take a stab at the structure when I add the compiled files